src/components/views/settings/devices/useOwnDevices.ts

```diff
@@ -91,10 +94,16 @@ export const useOwnDevices = (): DevicesState => {
     const userId = matrixClient.getUserId();
 
     const [devices, setDevices] = useState<DevicesState['devices']>({});
+    const [pushers, setPushers] = useState<DevicesState['pushers']>([]);
     const [isLoadingDeviceList, setIsLoadingDeviceList] = useState(true);
+    const [supportsMSC3881, setSupportsMSC3881] = useState(true); // optimisticly saying yes!
 
     const [error, setError] = useState<OwnDevicesError>();
 
+    matrixClient.doesServerSupportUnstableFeature("org.matrix.msc3881").then(hasSupport => {
```

Comment:
Shouldn't this be in a `useEffect`? It will set state every time the hook is called